### PR TITLE
Implement inso-extpipes-v3 deploy

### PR DESCRIPTION
**Pull request by pavelzubarev**
Implement Deploy command

### CODE REVIEW

**Review by chrfla-cognite — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -141,4 +141,9 @@ dmypy.json
 # vscode folder
 .vscode/
 
+# avoid automatic file add into git commit
+configs/
```
Comment (spex66) [inline .gitignore:145]: why is `configs/` in here?

**Review by spex66 — APPROVED** [review]
one question about .gitignore

File: .gitignore (hunk 1 above)
Comment (chrfla-cognite) [reply .gitignore:145]: added exceptions for the example files:

# Except the following files
!configs/example-config-extpipesv2.yml
!configs/example-config-extpipesv2to3.yml
!configs/example-config-extpipesv3.yml
!configs/example-config-extpipesv3-minimum.yml

Comment (chrfla-cognite) [reply .gitignore]: updated to 

!configs/example-config-extpipes*

**Pull request merged**